core/base/abstractTriangulation/AbstractTriangulation.h

```diff
@@ -3479,6 +3488,17 @@ namespace ttk {
       return 0;
     }
 
+    virtual inline int getCellVTKIDInternal(const int &ttkId,
+                                            int &vtkId) const {
+#ifdef TTK_ENABLE_KAMIZE
```

Comment:
shouldn't it be `#ifndef TTK_ENABLE_KAMIKAZE`?
in KAMIKAZE mode, we skip the sanity checks.
plus careful with the typo on KAMIKAZE.

Reply:
Yes, indeed, sorry about that